src/js/domain.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+import { addHistory } from './history.js';
+
+export const setDomainEventListeners = (domain) => {
```

Comment:
`domain`変数が未使用になっているけどこれは正しい？
```suggestion
export const setDomainEventListeners = () => {
```

Reply:
copilotのせい